Kimseho_assignment/app.js

```diff
@@ -0,0 +1,73 @@
+require("dotenv").config();
+
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
+});
+
+app.post("/user_signup", async (req, res, next) => {
+  const { name, email, password, profile_image } = req.body;
+
+  await myDataSource.query(
+    `INSERT INTO users(
+          name,
+          email,
+          password,
+          profile_image)
+      VALUES (?,?,?,?);
+    `,
+    [name, email, password, profile_image]
+  );
+
+  res.status(201).json({ message: "userCreated!" });
+});
```

Comment:
세호님 ! signup POST API의 endpoint 를 조금 더 restful 하게 작성해주세요 !
- 해당 블로그 참고하셔서 restful의 개념이 무엇인지 생각하시면 좋을 거 같습니다 !
- 다음 주에 restfulAPI 세션이 있으니 가벼운 마음으로 고민해 보세요 !

```suggestion
app.post("/user/signup", async (req, res, next) => {
  const { name, email, password, profile_image } = req.body;

  await myDataSource.query(
    `INSERT INTO users(
          name,
          email,
          password,
          profile_image)
      VALUES (?,?,?,?);
    `,
    [name, email, password, profile_image]
  );

  res.status(201).json({ message: "userCreated!" });
});
```